From a399795c667a40e6d2ae42d988a82225d3e38924 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=E9=BB=84=E6=B6=9B?= <huangtao@rock-chips.com>
Date: Tue, 25 Feb 2014 17:08:52 +0800
Subject: [PATCH] pwm-backlight: Remove unused variable

I forgot to remove this during earlier cleanup patches and only checked
various builds for errors, not warnings.

Reported-by: kbuild test robot <fengguang.wu@intel.com>
Signed-off-by: Thierry Reding <thierry.reding@gmail.com>
---
 drivers/video/backlight/pwm_bl.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/video/backlight/pwm_bl.c b/drivers/video/backlight/pwm_bl.c
index 3969bab1c32b..40b130fe1c6e 100644
--- a/drivers/video/backlight/pwm_bl.c
+++ b/drivers/video/backlight/pwm_bl.c
@@ -43,8 +43,6 @@ struct pwm_bl_data {
 
 static void pwm_backlight_power_on(struct pwm_bl_data *pb, int brightness)
 {
-	int duty_cycle, err;
-
 	if (pb->enabled)
 		return;
 
-- 
2.35.3

